Geometry/HGCalCommonData/data/hgcalwafer/v16p/hgcalpos.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<DDDefinition>
+
+<PosPartSection label="hgcalpos.xml">
+  <PosPart copyNumber="1">
+    <rParent name="cms:CMSE"/>
+    <rChild name="hgcalwafer:HGCalEEWafer0Fineb0"/>
+    <Translation x="-2.5*cm" y="-2.5*cm" z="0.0*fm"/>
```

Comment:
Just wondering: Is this file generated automatically? If not, why would we want to use "fm", that is femtometers (1e-15 meters), as a unit of measure for detector positions? Do we ever have fm accuracy in positioning detector components? Why not use `"0.0*mm"` instead of `"0.0*fm"`? Or maybe `"0.0*cm"` to be consistent with the units for `y`.

Reply:
It is left over from earlier usage. Shall us 0.0*mm in future